package.json

```diff
@@ -42,20 +42,19 @@
     "*.{js,jsx,ts,tsx,json,yaml,yml}": "yarn format:write"
   },
   "peerDependencies": {
-    "ethers": "^5.6.8"
+    "ethers": "^6.6.2"
   },
   "devDependencies": {
     "@commitlint/cli": "^17.0.2",
     "@commitlint/config-conventional": "^17.0.2",
     "@istanbuljs/nyc-config-typescript": "^1.0.2",
     "@looksrare/contracts-exchange-v1": "^1.2.0",
     "@looksrare/contracts-exchange-v2": "^0.1.2",
-    "@nomiclabs/hardhat-ethers": "^2.0.6",
-    "@nomiclabs/hardhat-waffle": "^2.0.3",
+    "@nomiclabs/hardhat-ethers": "3.0.0-beta.0",
```

Comment:
3.0.0-beta.0 is the latest version, I expect they will release the 3.0.0 version in the coming weeks, according to their git [history](https://github.com/NomicFoundation/hardhat/tree/main/packages/hardhat-ethers).

Reply:
https://github.com/NomicFoundation/hardhat/blob/main/packages/hardhat-ethers/package.json#L3
The latest version on the repository, is 3.0.4, but somehow it's not yet published to NPM
<img width="999" alt="image" src="https://github.com/LooksRare/sdk-v2/assets/98596804/78d24248-0ce5-4dc0-9756-87b9938ee1b0">

Reply:
I upgraded this to `@nomicfoundation/hardhat-ethers:^3.0.4` and this is no longer a problem.